src/arraymancer/laser/tensor/initialization.nim

```diff
@@ -13,7 +13,13 @@ import
   ../private/nested_containers,
   ./datatypes
 # Standard library
-import std / [typetraits, sequtils]
+import std / [typetraits, sequtils, sets]
+
+# The folling export is needed to avoid an compilation error in
+# algorithms.nim/intersection() when running the test_algorithms test:
+# `Error: type mismatch - Expression: items(s1)`
```

Comment:
```suggestion
# The following export is needed to avoid a compilation error in
# algorithms.nim/intersection() when running the test_algorithms test:
# `Error: type mismatch - Expression: items(s1)`
# (Alternative: could use `bind sets.items` in `intersection` and `setDiff`)
```